app/analysis/edit/controller.js

```diff
@@ -86,14 +86,16 @@ export default class AnalysisEditController extends Controller.extend(
 
   @task
   *intersection() {
-    const res = yield this.fetch.fetch("/api/v1/reports/intersection", {
-      method: "GET",
-      data: {
+    const res = yield this.fetch.fetch(
+      `/api/v1/reports/intersection?${new URLSearchParams({
```

Comment:
Adding a body to a `GET` request is not spec conform. That's why we should append additional query parameters to the URL string instead.